onnxruntime/test/providers/coreml/coreml_basic_test.cc

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+#if !defined(ORT_MINIMAL_BUILD) || defined(ORT_EXTENDED_MINIMAL_BUILD)
```

Comment:
is this necessary? looks like this file is only included by cmake if onnxruntime_USE_COREML is set, and there's a new check in this PR:
```
if (onnxruntime_USE_COREML)
  if (onnxruntime_MINIMAL_BUILD AND NOT onnxruntime_EXTENDED_MINIMAL_BUILD)
    message(FATAL_ERROR "CoreML EP can not be used in a basic minimal build. Please build with '--minimal_build extended'")
  endif()
  ...
```

Reply:
Yes, I think we can remove this, maybe the one in nnapi_basic_test.cc as well